app/static/app/js/components/SharePopup.jsx

```diff
@@ -83,7 +84,7 @@ class SharePopup extends React.Component{
                   (this.state.showQR ? "btn-primary " : "btn-default ") +
                   (!this.state.task.public ? "hide" : "")}
                 onClick={this.toggleQRCode}>
-                  <i className="fa fa-qrcode"></i> QR
+                  <i className="fa fa-qrcode"></i> {_("QR")}
```

Comment:
```suggestion
                  <i className="fa fa-qrcode"></i> {_("QR code")}
```
Otherwise it would be "Quick response" https://en.wikipedia.org/wiki/QR_code (?)